src/systems/physics/Physics.cc

```diff
@@ -2003,31 +2039,49 @@ void PhysicsPrivate::UpdateCollisions(EntityComponentManager &_ecm)
       [&](const Entity &_collEntity1, components::Collision *,
           components::ContactSensorData *_contacts) -> bool
       {
+        msgs::Contacts contactsComp;
         if (entityContactMap.find(_collEntity1) == entityContactMap.end())
         {
           // Clear the last contact data
           *_contacts = components::ContactSensorData();
-          return true;
-        }
-
-        const auto &contactMap = entityContactMap[_collEntity1];
-
-        msgs::Contacts contactsComp;
 
-        for (const auto &[collEntity2, contactData] : contactMap)
-        {
+          // If there's an empty ContactSensorData, it won't be updated in the
+          // GUI ECM and there could be remaining contacts which shouldn't
+          // exist anymore. We insert dummy data in the empty ContactSensorData
+          // in order to avoid this problem.
```

Comment:
Is this still needed now that the GUI knows of removed components? (#272)

Reply:
Hmm for some reason when I delete that code, the forces are only displayed once and then they don't get updated.

Also, when I try to disable a collision by requesting the `disable_collision` service, forces keep showing up most of the times. I'll take a look at the `SerializedState` msg to see what's going on.